testgrid/cmd/configurator/prow.go

```diff
@@ -41,6 +41,7 @@ const testgridNumColumnsRecentAnnotation = "testgrid-num-columns-recent"
 const testgridAlertStaleResultsHoursAnnotation = "testgrid-alert-stale-results-hours"
 const testgridNumFailuresToAlertAnnotation = "testgrid-num-failures-to-alert"
 const testgridDaysOfResultsAnnotation = "testgrid-days-of-results"
+const testgridShortTextMetric = "testgrid-short-text-metric"
```

Comment:
Depending on the use case, it may be best to make the annotation more descriptive.

"Short Text" has a clearer meaning in the TestGrid space (ie. The 1-5 characters displayed inside a cell), but we might want, like, "testgrid-metric-in-cell-text" or "testgrid-display-metric"

Reply:
This is part of [Issue 1914](https://github.com/knative/test-infra/issues/1914). All the Prow Jobs currently have TestGrid configured [here](https://github.com/knative/test-infra/blob/master/config/prod/prow/testgrid/testgrid.yaml). Some of these Prow Jobs have the field "short_text_metric" set, for example [this one](https://github.com/knative/test-infra/blob/8f5dc4814ac37f9d504bd6872673a7d1bdbb0f5d/config/prod/prow/testgrid/testgrid.yaml#L120). The [other PR](https://github.com/knative/test-infra/pull/2511) I'm working on is able to use the existing Configurator annotations for all of these TestGrid configs with the exception of this "short_text_metric" field, support for which I'm hoping this PR will enable.

Reply:
Hmm. I think it's okay since it's the same as the proto field, but I'm also okay with a clearer name along the lines of what Sean suggested.

Reply:
Per offline conversation renaming this to "testgrid-in-cell-metric"